code_samples/paging_predicate_sample/main.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const { Client, Predicates } = require('hazelcast-client');
+
+class Comparator {
+    constructor() {
+        this.factoryId = 66;
+        this.classId = 103;
+    }
+
+    sort(a, b) {
+        if (a[1] > b[1]) {
```

Comment:
Can you remove the method body and add a comment saying that the actual sorting happens on the member side and nothing is required here?